### PR TITLE
Feat/navigation

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Added Navigation
Extended utility styles by adding flex utility

### CODE REVIEW

**Review by sahiba0915 — COMMENTED** [review]
looks good to me

File: Components/nav.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    width: 1.3rem;
+    height: 1.3rem;
+    font-size: 1.1rem;
+  }
+  .search-bar{
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+    position: relative;
+    background-color:  #f7f7fd;
```
Comment (arjunbharti) [inline Components/nav.css:67]: Use color variables

File: Components/nav.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    padding: 0.5rem 0.7rem;
+    background: transparent;
+    border: none;
+    font-size: 1rem;
+    color: var(--dark-secondary-text-color);
+  }
+  .search-bar-input{
+    padding: 0.5rem 0.7rem;
+    border: none;
+    background-color:  #f7f7fd;
```
Comment (arjunbharti) [inline Components/nav.css:79]: Use color variables

**Review by arjunbharti — APPROVED** [review]
Looks good to me. Just a little suggestion - use color variables.

**Pull request merged**